api/HTMLAreaElement.json

```diff
@@ -574,54 +574,6 @@
             "deprecated": false
           }
         }
-      },
-      "type": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLAreaElement/type",
-          "support": {
-            "chrome": {
-              "version_added": true
-            },
-            "chrome_android": {
-              "version_added": true
-            },
-            "edge": {
-              "version_added": "12"
```

Comment:
Confirmed wrong in http://mdn-bcd-collector.appspot.com/tests/api/HTMLAreaElement, it's not in Edge 18.